### ISSUE TITLE
BUG: Latest PHP syntax not being supported:  ??

### ISSUE THREAD

**Issue by EliW** [AceIDE/editor#25]
When attempting to use the ?? coalesce operator, it claims incorrect syntax, and refuses to save the code.

For example:
`$output->address_1 = $data['raw']['charge']['source']['address_line1'] ?? $output->data['raw']['token']['card']['address_line1'] ?? '';`

**Comment by shanept** [comment]
This probably just requires an update of ace editor. Can you try a composer update to see if this fixes the issue?

**Comment by shanept** [comment]
Hi EliW,

I have upgraded the version of Ace being used in AceIDE and I can confirm that your specific code snippet is no longer flagged as a syntax error.

However, you will run in to issues with the inbuilt PHP parser for the time being. It currently doesn't seem to support PHP 7. For this reason, I've added a filter to allow you to disable the inbuilt parser, as follows:

`add_filter( 'aceide_parse_php', '__return_false' );`

Thanks for reporting this issue!
